### DISCUSSION TITLE
Openvslam as visual odometry

### DISCUSSION THREAD

**Discussion by manosmaroulis** [stella-cv/stella_vslam#67] [Q&A] [answered]
Hello, i want to ask if it makes sense to use openvslam as VO (visual odometry) , with a 360 camera, or is it too heavy for no reason. Also what tweaks could i do to lighten the algorithm in order to use it as VO. Thank you.

**Comment by ymd-stella** [comment, marked as answer]
Different applications require different performance, and the performance depends on the CPU.
Therefore, it is very difficult to give a general answer.
On wercker's CI machine, OpenVSLAM is able to handle 20 fps for monocular and 10 fps for stereo.

ref. https://app.wercker.com/srz-zumix/ci-specs/runs/build/6083fef0237ec90008522e92?step=6083ff07476e9000081f9887

As for the second question, it is even more difficult to answer. If the aforementioned level of performance is not achievable with a comparable CPU, please ask the question with detailed specs.
